docs/pages/application-access/cloud-apis/azure-aks-workload-id.mdx

```diff
@@ -0,0 +1,239 @@
+  commands. See the [Azure
+  documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+  for how to install the `az` CLI on your operating system.
+
+- `kubectl` and `helm` for AKS deployments.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+
+## Step 1/6. Create an Azure managed identity for Teleport Application Service
+
+Teleport Application Service requires a managed identity that can retrieve the
+client IDs of the managed identities for user access. This managed identity
+will be assigned as the default identity for the Kubernetes service account.
+
+Login to your Azure admin account with `az login` command if you haven't
+already, and prepare some environment variables for later steps:
+```code
+$ export SUBSCRIPTION="$(az account show --query id --output tsv)"
+$ export LOCATION="<Var name="eastus" />"
+$ export RESOURCE_GROUP="<Var name="myResourceGroup" />"
+$ export AKS_CLUSTER_NAME="<Var name="myASKCluster" />"
+$ export USER_ASSIGNED_IDENTITY_NAME="<Var name="teleport-azure-cli-aks-agent" />"
+```
+
+Now create the managed identity, and remember the client ID for a later
+step:
+```code
+$ az identity create --name "${USER_ASSIGNED_IDENTITY_NAME}" --resource-group "${RESOURCE_GROUP}" --location "${LOCATION}" --subscription "${SUBSCRIPTION}"
+$ export USER_ASSIGNED_CLIENT_ID="$(az identity show --resource-group "${RESOURCE_GROUP}" --name "${USER_ASSIGNED_IDENTITY_NAME}" --query 'clientId' -o tsv)"
+```
+
+Next, create a role with
+`Microsoft.ManagedIdentity/userAssignedIdentities/read` permission and assign
+it to the managed identity:
+```code
+$ cat > ${USER_ASSIGNED_IDENTITY_NAME}-role.json <<EOF
+{
+  "Name": "${USER_ASSIGNED_IDENTITY_NAME}-role",
+  "Description": "Role for Teleport Azure CLI Access on AKS",
+  "AssignableScopes": [ "/subscriptions/${SUBSCRIPTION}" ],
+  "Actions": [ "Microsoft.ManagedIdentity/userAssignedIdentities/read" ],
+  "notActions": []
+}
+EOF
+$ az role definition create --role-definition ./${USER_ASSIGNED_IDENTITY_NAME}-role.json
+$ az role assignment create --role "${USER_ASSIGNED_IDENTITY_NAME}-role" --scope "/subscriptions/${SUBSCRIPTION}" --assignee-object-id $(az identity show --name "${USER_ASSIGNED_IDENTITY_NAME}" --resource-group "${RESOURCE_GROUP}" --query principalId --output tsv) --assignee-principal-type ServicePrincipal
+```
+
+## Step 2/6. Configure the AKS cluster for Workload ID
+
+To use Microsoft Entra Workload ID, you need to enable OIDC issuer and Workload
+ID in your AKS cluster.
+```code
+$ az aks update -g "${RESOURCE_GROUP}" -n "{AKS_CLUSTER_NAME}" --enable-oidc-issuer --enable-workload-identity
+```
+
+Before using `kubectl`, make sure your local Kubernetes config is updated to
+access your AKS cluster:
+```code
+$ az aks get-credentials -n "${AKS_CLUSTER_NAME}" -g "${RESOURCE_GROUP}"
+```
+
+Create a Kubernetes service account and annotate it with the client ID of
+the managed identity created in the previous step:
+```code
+$ cat <<EOF | kubectl apply -f -
```

Comment:
It might make sense to separate the steps of creating a manifest and applying it so the user can review the YAML file. `Var`s can be a little tricky to use for readers who aren't familiar with them, so it'd be helpful to give the reader a chance to make sure spacing is correct etc..